### DISCUSSION TITLE
Integrating nocodb as a one click app

### DISCUSSION THREAD

**Discussion by eliagentili** [caprover/caprover#1118] [Ideas]
Hi there,
I don't have the competencies to that by myself, but I want to suggest integrating this (IMHO) amazing project:
[https://github.com/nocodb/nocodb](https://github.com/nocodb/nocodb)

It's an open-source project that emulates AirTable, and I think it can add real value to the CapRover community.

Thank You all in advance!

**Comment by githubsaturn** [comment]
Feel free to open an issue here: https://github.com/caprover/one-click-apps/issues
